.github/workflows/test.yml

```
@@ -35,17 +35,21 @@ jobs:
     - run: npm test
   tox:
     name: >-
-      ${{ matrix.env.TOXENV }}
+      ${{ matrix.tox_env }}
```

Let's use
```suggestion
      ${{ matrix.toxenv }}
```
to match the typical tox's terminology and env var naming.